src/kvirc/kvs/event/KviKvsEventTable.cpp

```diff
@@ -3986,6 +3986,26 @@ KviKvsEvent KviKvsEventManager::m_appEventTable[KVI_KVS_NUM_APP_EVENTS]=
 		"$0 = source nick\n"
 		"$1 = source username\n" \
 		"$2 = source hostname\n" \
-		"$3 = ban mask")
+		"$3 = ban mask"),
 
+	/*
+		@doc: onmehostchange
+		@type:
+			event
+		@title:
+			OnMeHostChange
+		@short:
+			Local user's visible host is changed
+		@parameters:
+			$0 = source server
+			$1 = new host
+		@window:
+			Console
+		@description:
```

Comment:
It's impossible, technically all IRCds support this, however, the only one that doesn't is OFTC.